DOCSP-32898 Enable Natural Language Querying (#551)
mongodb/docs-compass: source/query-with-natural-language/enable-natural-language-querying.txt

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1

You can enable Compass natural language querying by toggling the 

> :guilabel:`AI Query Generator` setting. Natural language queries can 

Reviewer: ```suggestion
:guilabel:`Use Generative AI` setting. Natural language queries can 
```

be a helpful starting point and assist you in learning to write MongoDB 
queries.

About this Task
---------------

Alternatively, you can enable natural language querying by clicking the 
:guilabel:`Log in to Atlas to enable` button within the 
:guilabel:`Use natural language to generate queries and pipelines` modal. 
This modal displays after updating Compass to a version that supports 
natural language querying and clicking the :guilabel:`Generate` 
button on the query bar.

Before you Begin
----------------

> To enable Compass natural language querying you must connect to an 

Reviewer: ```suggestion
To enable the generation of queries and aggregations in Compass, you must connect to an 
```

Atlas deployment. For details on how to get a free Atlas account and 
deploy a free M0 cluster, see :ref:`atlas-getting-started`.

Steps
-----

.. procedure::
   :style:  connected

   .. step:: Open settings

      .. list-table::
         :header-rows: 1
         :widths: 30 50 20

         * - Operating System
           - Description
           - Keyboard Shortcut

         * - Windows / Linux
           - In the top menu bar, click :guilabel:`Edit`.
           - ``Ctrl`` + ``,``

         * - macOS
           - In the top menu bar, click :guilabel:`MongoDB Compass`.
           - ``⌘`` + ``,``

   .. step:: Connect to your Atlas account

      I. Select :guilabel:`Feature preview`
      #. Click :guilabel:`Log in with Atlas`

         A web browser displays the
         `Atlas login page <https://account.mongodb.com/account/login>`_.

      #. Complete the login procedure

         Once completed a :guilabel:`Logged in with Atlas account`
         message displays. 

>    .. step:: Enable the :guilabel:`AI Query Generator` feature

Reviewer: ```suggestion
   .. step:: Enable the :guilabel:`Use Generative AI` feature
```

>       I. Click the toggle for :guilabel:`Use AI to generate queries with a 

Reviewer: At this point in time, a user is directed back from Atlas to Compass, where the "Data Processing Agreement" modal displays. When they accept it, the toggle is automatically activated, so there is no need for them to click the toggle.

Author: Alright I have removed this from the steps.
